pr-request-template.md

```diff
@@ -0,0 +1,77 @@
+# Pull Request Template
+
+Thank you for your contribution to Alumnium!
+Please fill out this template completely to help us understand and review your PR efficiently.
+
+## Summary
+What is this pull request about?
+Provide a brief description of what the PR does, the problem it solves, and any context around the implementation.
+
+## Motivation
+Why is this change being made?
+Describe the use case, problem, or opportunity that this PR addresses.
+Explain how this change contributes to the project's goals or improves functionality.
+
+## Related Issue(s)
+If this PR fixes or is related to an open issue, link it here.
+Example: Closes #123
+
+Closes #
+
+## Type of Change
+Mark the relevant options.
+
+- [ ] Bug fix  
+- [ ] New feature  
+- [ ] Documentation update  
+- [ ] Test coverage improvement  
+- [ ] Refactoring  
+- [ ] CI/CD enhancement
+- [ ] Breaking change
+- [ ] Other (please specify):
```

Comment:
Let's join some of the options, so it's easier to choose. WDYT about this list:
```suggestion
- [ ] Bug fix  
- [ ] New feature (non-breaking changes, test coverage, refactoring)
- [ ] Breaking change (fix, refactoring, or feature that would cause existing functionality to change)
- [ ] Cleanup (documentation, formatting)
```